gobcore/events/import_events.py

```diff
@@ -0,0 +1,196 @@
+"""GOB Events
+
+Each possible event in GOB is defined in this module.
+The definition includes:
+    name - the name of the event, e.g. ADD
+    timestamp - the name of the timestamp attribute in the GOB entity that tells when this event has last been applied
+    get_modification - method to
+
+todo: The delete and confirm actions contain too much data. Contents can be left empty
+    A deletion or confirmation simple specifies source and sourceid in a collection
+    See also the examples in the action classes
+
```

Comment:
If we leave this as a todo my remark in __init__ about Delete events can be removed

Reply:
I'll leave a todo